dptb/entrypoints/train.py

```
@@ -90,6 +91,35 @@ def train(
+    # Validate and process orbital files in basis
+    if jdata.get("common_options") and jdata["common_options"].get("basis"):
+        orbital_files_content = {}
+        for elem, value in jdata["common_options"]["basis"].items():
+            if isinstance(value, str) and os.path.isfile(value):
+                # strict check for e3tb method
+                # Check if model_options exists and has prediction method
+                model_opts = jdata.get("model_options", {})
+                pred_opts = model_opts.get("prediction", {})
+                # normalize might handle defaults, but safely check here
+                if pred_opts and pred_opts.get("method", "e3tb") != "e3tb":
+                     raise ValueError(f"Orbital files in 'basis' are only supported for the 'e3tb' method. Found method: {pred_opts.get('method')}")
```

Ruff (0.15.0)

[warning] Avoid specifying long messages outside the exception class

(TRY003)